src/homework19/GaragePage/GaragePage.js

```diff
@@ -0,0 +1,12 @@
+import BasePage from "../BasePage.js";
+
+
+export default class GaragePage extends BasePage{
+    constructor(page) {
+        super(page, "/panel/garage");
+        this.titleGaragePage =  page.locator('h1', {hasText: 'Garage'});
+        this.profileBtnInHeader = page.getByRole('button', {name: 'My profile'});
```

Comment:
this button is visible not only on this page

I would create Header component

Reply:
I created Header component